openlibrary/plugins/openlibrary/js/bulk-tagger/BulkTagger.js

```diff
@@ -345,6 +345,7 @@ export class BulkTagger {
             break
         }
 
+        this.highlightMenuOption(menuOption);
```

Comment:
```suggestion
        menuOption.stage()
```
Could you add a `stage` method to `MenuOption`?  This should add the `selected-tag--staged` class to the menu option.